content/docs/04-clusters/06-cluster-management/02.5-certificate-management.md

```diff
@@ -0,0 +1,421 @@
+---
+title: "Certificate Management"
+metaTitle: "Certificate Management"
+metaDescription: "Learn how to manage the cluster certificates through Palette."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Overview
+
+Palette installs Kubernetes through the tool, [kubeadm](https://kubernetes.io/docs/reference/setup-tools/kubeadm). As a result, all deployed clusters include auto-generated Public Key Infrastructure (PKI) certificates created by kubeadm. We recommend you review the [PKI certificates and requirement](https://kubernetes.io/docs/setup/best-practices/certificates) documentation from the official Kubernetes documentation to learn more about the auto-generated certificates and to understand their purpose better.
+
+This reference page focuses on how to renew the PKI certificates through Palette. You have two options for how you can renew the cluster PKI certificates:
+
+  <br />
+
+  * Automatic Certificate Renewal
+
+
+  * Manual Certificate Renewal
+
+
+
+<InfoBox>
+
+Certificates created by kubeadm expire after 365 days. The Root Certificate Authority (CA) is valid for 3652 days or ten years.
+
+</InfoBox>
+
+
+You can learn more about each option in the following sections. 
+
+
+
+# Automatic Certificate Renewal
+
+When you upgrade the control plane on a cluster, the PKI certificates are automatically updated. Upgrading a cluster's Kubernetes version, whether a minor patch or a major release, results in renewed PKI certificates. This is the method that requires the least user actions when it comes to renewing PKI certificates. We recommend regularly updating your clusters to stay up-to-date with security fixes and best practices. By keeping your host cluster up-to-date, you prevent the scenario of PKI certificates from reaching their one-year expiration date.
```

Comment:
```suggestion
When you upgrade the control plane on a cluster, the PKI certificates are automatically updated. Upgrading a cluster's Kubernetes version, whether a minor patch or a major release, results in renewed PKI certificates. This is the method that requires the least user actions when it comes to renewing PKI certificates. We recommend regularly updating your clusters to stay up-to-date with security fixes and best practices. By keeping your host cluster up-to-date, you prevent the scenario of PKI certificates from reaching their one-year expiration date.
```

To make this more concise.

Reply:
I'm declining this as we want to call out this method has the least actions.